comid/example_test.go

```diff
@@ -367,7 +419,111 @@ func Example_decode_JSON() {
 					}
 				]
 			}
-		]
+		],
+        "conditional-endorsement-series": [
+            {
+                "statefulenv": {
+                    "environment": {
+                        "class": {
+                            "id": {
+                                "type": "oid",
+                                "value": "2.5.2.8192"
+                            },
+                            "vendor": "ACME Ltd.",
+                            "model": "RoadRunner",
+                            "layer": 0,
+                            "index": 1
+                        },
+                        "instance": {
+                            "type": "ueid",
+                            "value": "At6tvu/erQ=="
+                        },
+                        "group": {
+                            "type": "uuid",
+                            "value": "31fb5abf-023e-4992-aa4e-95f9c1503bfa"
+                        }
+                    },
+                    "measurements": [
+                        {
+                            "key": {
+                                "type": "uuid",
+                                "value": "31fb5abf-023e-4992-aa4e-95f9c1503bfa"
+                            },
+                            "value": {
+                                "svn": {
+                                    "type": "exact-value",
+                                    "value": 2
+                                },
+                                "digests": [
+                                    "sha-256-32;q83vAA==",
+                                    "sha-256-32;/////w=="
+                                ],
+                                "flags": {
+                                    "is-secure": false,
+                                    "is-debug": true
+                                },
+                                "raw-value": {
+                                    "type": "bytes",
+                                    "value": "AQIDBA=="
+                                },
+                                "raw-value-mask": "/////w==",
+                                "mac-addr": "02:00:5e:10:00:00:00:01",
+                                "ip-addr": "2001:db8::68",
+                                "serial-number": "C02X70VHJHD5",
+                                "ueid": "At6tvu/erQ==",
+                                "uuid": "31fb5abf-023e-4992-aa4e-95f9c1503bfa"
+                            }
+                        }
+                    ]
+                },
+                "series": {
+                    "Values": [
+                        {
+                            "selection": [
```

Comment:
I don't think the selection needs to repeat the values in the condition.
The use case should have multiple entries in the series. The selection attributes should be the same for each series case. Maybe the series semantics are being blended with the condition?

Reply:
ok, i will address this, thanks for looking into this!

Reply:
Resolved it now!